pages/linux/dm-tool.md

```diff
@@ -0,0 +1,20 @@
+# dm-tool
+
+> A tool to commnicate with the display manager. EG: lock currect session and show lightdm, list all open sessions, connect an existing x server to the display manager.
+> More informatioin: https://www.mankier.com/1/dm-tool.
```

Comment:
```suggestion
> More information: <https://www.mankier.com/1/dm-tool>.
```

Reply:
@godalming123, please prefer [manned.org](https://manned.org/dm-tool.1) when it's possible.

Reply:
Ok I won't necessarily use maned.org but will accept all of the other changes here on Monday night

Reply:
Unless there is a valid reason, the manned page for this must be used. I see no difference between the current link and the one on manned.org